app/main/posts/detail/detail.html

```diff
@@ -155,5 +155,6 @@ <h3 class="listing-heading" translate="app.tasks">Tasks</h3>
 
     <post-add-form post="post" ng-if="!post.form && post.allowed_privileges.indexOf('update') !== -1"></post-add-form>
     <post-messages post="post" ng-if="post.contact.id"></post-messages>
+    <ush-logo></ush-logo>
```

Comment:
Unfortunately, it can't live at the top as the css is hierarchy dependent so it has to go here. I asked @brandonrosage about the ability to move it and he said that he can look into it but that it relates to a core approach(I may be mangling and completely misrepresenting what he meant).